example/bwmf/block_parameter.go

```diff
@@ -0,0 +1,137 @@
+package bwmf
+
+import (
+	"fmt"
+
+	pb "github.com/taskgraph/taskgraph/example/bwmf/proto"
+	op "github.com/taskgraph/taskgraph/op"
```

Comment:
Got it. Sounds like this isn't right for a package name. Anyway, we can fix this later.